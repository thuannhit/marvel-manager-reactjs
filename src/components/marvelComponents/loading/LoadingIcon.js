import React from 'react';
import './style.css';
import PropTypes from 'prop-types';
import { withStyles } from '@material-ui/core/styles';
import CircularProgress from '@material-ui/core/CircularProgress';

const styles = theme => ({
  progress: {
    margin: theme.spacing.unit * 2,
  },
});

function CircularIndeterminate(props) {
  const { classes } = props;
  return (
    <div>
        {/* <div className="text-center h1 Loading">
          Loading<span>.</span><span>.</span><span>.</span>
        </div> */}
        <CircularProgress className={classes.progress +" text-center"} color="secondary" />
    </div>
  );
}

CircularIndeterminate.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(CircularIndeterminate);
