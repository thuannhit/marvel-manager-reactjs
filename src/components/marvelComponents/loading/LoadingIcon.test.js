import React from 'react';
import LoadingIcon from './LoadingIcon';
import renderer from 'react-test-renderer';

it('LoadingIcon renders correctly', () => {
  const tree = renderer
    .create(<LoadingIcon/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});