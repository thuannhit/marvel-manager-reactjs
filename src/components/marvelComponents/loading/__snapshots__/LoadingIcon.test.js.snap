// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIcon renders correctly 1`] = `
<div>
  <div
    className="MuiCircularProgress-root-2 MuiCircularProgress-colorSecondary-6 MuiCircularProgress-indeterminate-4 CircularIndeterminate-progress-1 text-center"
    role="progressbar"
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg-7"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle-8 MuiCircularProgress-circleIndeterminate-10"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
